src/main/fc/cli.c

```diff
@@ -374,8 +374,8 @@ typedef enum {
 #ifdef BLACKBOX
     TABLE_BLACKBOX_DEVICE,
 #endif
-    TABLE_CURRENT_SENSOR,
-    TABLE_BATTERY_SENSOR,
+    TABLE_CURRENT_METER,
```

Comment:
Although this distinction is logical, it doesn't fully align with other parts of the code. So, for example, the gyro in the `sensors` directory returns calibrated, scaled and filtered data.

I don't suggest you change the code, but perhaps you could update that description to reflect this.